### ISSUE TITLE
Wrapper behavior is unexpected: please update log message

### ISSUE THREAD

**Issue by colindembovsky** [hashicorp/setup-terraform#85]
Hi team - thank you for your work on this action! I have one piece of feedback: the wrapper behavior is unexpected. Here's a snippet of a workflow that I was attempting:

```
steps:
  - name: Install TF
    uses: hashicorp/setup-terraform@v1
  - name: Get webappURL
    run: |
      $url=$(terraform output -raw webappURL)
      echo "::set-output name=webapp_url::$url"
```

Instead of `url` being `cdtailwindgha-dev.azurewebsites.net` (the expected value from the `terraform output`), the value was set to:
```
[command]/home/runner/work/_temp/d6cc4736-dad6-4784-b42d-6c80ea5e10da/terraform-bin output -raw slotURL cdtailwindgha-dev.azurewebsites.net::debug::Terraform exited with code 0. ::debug::stdout: cdtailwindgha-dev.azurewebsites.net ::debug::stderr: ::debug::exitcode: 0 ::set-output name=stdout::cdtailwindgha-dev.azurewebsites.net ::set-output name=stderr:: ::set-output name=exitcode::0
```

Admittedly, _without reading the docs on the setup-terraform action_ I expected the above to work - but the output `webapp_url` was set to the output of the wrapper. I thrashed for hours trying to figure out what was going on.

Eventually I was pointed to this behavior by some helpful folks at GitHub.

Can I request that you put a log message stating that the wrapper is being used and a link to the readme? That way users like me who don't read the docs initially can be prompted to do so!

**Comment by allanmc** [comment]
I have the same issue. As a workaround, try disabling `terraform_wrapper` as described here: https://github.com/hashicorp/setup-terraform

**Comment by austinvalle** [comment]
This issue will be fixed in an upcoming `v3.0.0` release next week. Please see [this comment](https://github.com/hashicorp/setup-terraform/issues/167#issuecomment-1783310845) for more info
